lib/hx509/hxtool.c

```diff
@@ -2898,13 +2898,14 @@ acert1_kus(struct acert_options *opt,
 static time_t
 ptime(const char *s)
 {
-    struct tm at_tm;
+    struct tm at_tm = {};
     char *rest;
     int at_s;
 
     if ((rest = strptime(s, "%Y-%m-%dT%H:%M:%S", &at_tm)) != NULL &&
         rest[0] == '\0')
         return mktime(&at_tm);
+    at_tm = (struct tm) {};
```

Comment:
Does MSVC support this?

Reply:
@jaltman says "no".  Can you use `memset()` instead?

Reply:
OK.